homeassistant/components/water_heater/__init__.py

```diff
@@ -49,6 +49,7 @@
 
 STATE_ECO = "eco"
 STATE_ELECTRIC = "electric"
+STATE_MANUAL = "manual"
```

Comment:
This PR is changing our entity model. Before we can review this PR there needs to be approval in a discussion in our architecture repository.

https://github.com/home-assistant/architecture/discussions

https://developers.home-assistant.io/docs/core/entity#changing-the-entity-model

Note that getting approval in an architecture issue can take a long time.

Reply:
Thank you, I just open a discussion: https://github.com/home-assistant/architecture/discussions/873